client/state/happychat/test/selectors.js

```diff
@@ -276,6 +277,17 @@ describe( 'selectors', () => {
 	} );
 
 	describe( '#getGroups()', () => {
+		let _window; // Keep a copy of the original window if any
```

Comment:
Not a big deal but looping in @gwwar since this might have a cascading effect on other tests that try to use this function. Is there a clean way you can think of to remove [this](https://github.com/Automattic/wp-calypso/blob/fix/happychat/at-sites-considered-jetpack/client/lib/automated-transfer/index.js#L19) so that we can use it in tests without the boilerplate `global.window` assignments and so that it won't break server side rendering as your comment suggests?

My thoughts were that the SSR `global.window` check could be moved to the component thats actually using the `isAtEnabled` function.